packages/plugins/other/visitor-plugin-common/src/client-side-base-visitor.ts

```diff
@@ -333,12 +340,14 @@ export class ClientSideBaseVisitor<
       return JSON.stringify(gqlObj);
     }
     if (this.config.documentMode === DocumentMode.string) {
-      return '`' + doc + '`';
+      return '`' + this.config.stripIgnoredCharacters ? stripIgnoredCharacters(doc) : doc + '`';
     }
 
     const gqlImport = this._parseImport(this.config.gqlImport || 'graphql-tag');
 
-    return (gqlImport.propName || 'gql') + '`' + doc + '`';
+    return (gqlImport.propName || 'gql') + '`' + this.config.stripIgnoredCharacters
```

Comment:
This change impact all configurations of `documentMode`, while this new feature should only apply to `documentMode: "string"`

Reply:
@charlypoly Well, this is expected though. This feature should apply to any `documentMode` that spits out strings, which includes both `string` and `graphQLTag` — `graphQLTag` is basically a string with a `gql` tab in front of it, so we should remove redundant characters for that string too.

And so this is what's happening here.